libs/state/selections/src/lib/operators/select.ts

```diff
@@ -150,11 +202,25 @@ export function select<T,
  * @internal
  */
 export function select<T>(
-  ...opOrMapFn: OperatorFunction<T, any>[] | string[]
+  ...opOrMapFn:
+    | OperatorFunction<T, any>[]
+    | string[]
+    | [string, (val: any) => any]
+    | [string[], (slice: any) => any]
 ): OperatorFunction<T, any> {
   return (state$: Observable<T>) => {
     if (!opOrMapFn || opOrMapFn.length === 0) {
       return state$.pipe(stateful());
+    } else if (isStringAndFunctionTupleGuard(opOrMapFn)) {
+      return state$.pipe(
+        stateful(pluck(opOrMapFn[0])),
+        stateful(map(opOrMapFn[1]))
```

Comment:
can we combine those two somehow? stateful itself applies quite some operators and I guess we do not need to duplicate all of them

Reply:
Good point :+1: Fixed it.